opennms-webapp/src/main/webapp/WEB-INF/jsp/admin/editCategory.jsp

```diff
@@ -73,10 +73,14 @@ function populateOptGroupFromList(elementName, list) {
 		optgroupElem.remove(0);
 	}
 	
-	for (var nodeId in list) {
+	for (const nodeSpec of list) {
```

Comment:
The (somewhat) efficient way to do this in es5 is:

```
for (var i=0, len=list.length, nodeSpec; i < len; i++) {
    nodeSpec = list[i];
```

Reply:
Thanks. Glad I asked :) I'll rework it to do this instead.